specification/professionalservice/resource-manager/Microsoft.ProfessionalService/preview/2023-07-01-preview/examples/Operation/GetOperation.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
New service must use RPaas
https://armwiki.azurewebsites.net/rpaas/overview.html

Reply:
We got an exception from Vlad Joanovic & Evan Hissey to not use RPaaS here at this time. (This is because the backend of ProfessionalServices RP is shared with the existing SaaS RP, and we will migrate them both together to RPaaS in the near future after ProfessionalServices RP initial onboarding)